util/libcrypto.num

```diff
@@ -5688,3 +5688,8 @@ i2d_ATTRIBUTES_SYNTAX                   ?	3_4_0	EXIST::FUNCTION:
 ATTRIBUTES_SYNTAX_free                  ?	3_4_0	EXIST::FUNCTION:
 ATTRIBUTES_SYNTAX_new                   ?	3_4_0	EXIST::FUNCTION:
 ATTRIBUTES_SYNTAX_it                    ?	3_4_0	EXIST::FUNCTION:
+d2i_USER_NOTICE_SYNTAX                  ?	3_4_0	EXIST::FUNCTION:
+i2d_USER_NOTICE_SYNTAX                  ?	3_4_0	EXIST::FUNCTION:
+USER_NOTICE_SYNTAX_free                 ?	3_4_0	EXIST::FUNCTION:
+USER_NOTICE_SYNTAX_new                  ?	3_4_0	EXIST::FUNCTION:
+USER_NOTICE_SYNTAX_it                   ?	3_4_0	EXIST::FUNCTION:
```

Comment:
Hmm... should these be renamed to OSSL_USER_NOTICE? And that would apply also to the ATTRIBUTES_SYNTAX-> OSSL_ATTR_SYNTAX ? There is a potential name clash here.

Reply:
I think you're right, but `ATTRIBUTES_SYNTAX` was already merged in. I'll change it in this PR if you want, but it seems out of place to do it here.

Reply:
Fixed.